frontend/packages/console-app/src/components/user-preferences/UserPreferencePage.scss

```
@@ -0,0 +1,16 @@
+.co-user-preference-page-content {
+  display: flex;
+  flex-direction: row;
+  background-color: var(--pf-global--BackgroundColor--100);
+  height: 100%;
```

Don't force the height, I believe you want ask for a minimum height. ;)

```suggestion
  min-height: 100%;
```